webpack/assets/javascripts/react_app/routes/RegistrationCommands/RegistrationCommandsPage/components/Actions.js

```diff
@@ -0,0 +1,59 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Link } from 'react-router-dom';
+
+import {
+  ActionGroup,
+  Button,
+  FormHelperText,
+  FormGroup,
+} from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons/dist/js/icons/exclamation-circle-icon';
```

Comment:
Looks like this broke the build, this isn't exposed by foreman-js vendor: https://github.com/theforeman/foreman-js/blob/master/packages/vendor-core/lib/modules.js
However I think it should be possible to use 
```
import { ExclamationCircleIcon } from '@patternfly/react-icons';
```
instead

Reply:
BTW, in the past we had a lint rule that would yell about this, not sure what happened to it.

Reply:
opened a fix in https://github.com/theforeman/foreman/pull/8467

I also remember a linting rule that used to yell about it,
though maybe it was more for something like `import sortBy from 'lodash/SortBy'`?
Today you can do just `import {sortBy} from lodash` btw :D